pkg/prometheus/store.go

```diff
@@ -83,13 +83,15 @@ func AddRemoteReadsToStore(ctx context.Context, store *assets.StoreBuilder, name
 func AddAlertmanagerEndpointsToStore(ctx context.Context, store *assets.StoreBuilder, namespace string, ams []monv1.AlertmanagerEndpoints) error {
 	for i, am := range ams {
 		if err := store.AddBasicAuth(ctx, namespace, am.BasicAuth); err != nil {
-			return fmt.Errorf("alertmanager %d: %w", i, err)
-		}
-		if err := store.AddSafeAuthorizationCredentials(ctx, namespace, am.Authorization, fmt.Sprintf("alertmanager/auth/%d", i)); err != nil {
-			return fmt.Errorf("alertmanager %d: %w", i, err)
-		}
-		if err := store.AddSigV4(ctx, namespace, am.Sigv4, fmt.Sprintf("alertmanager/auth/%d", i)); err != nil {
-			return fmt.Errorf("alertmanager %d: %w", i, err)
+			if err := store.AddBasicAuth(ctx, namespace, am.BasicAuth); err != nil {
```

Comment:
Not sure how this change relates to the rest of the PR...?

Probably by accident?

Reply:
Yeah, I probably mixed up the branches.

Reply:
Nothing seems to have actually changed though, is it fine to keep this in? I squashed all the commits I had made, so IDK if I can revert it. Sorry!!